packit_service/worker/jobs.py

```diff
@@ -107,22 +107,36 @@ def get_config_for_handler_kls(
     :param package_config: we pick the JobConfig from this package_config instance
     :return: JobConfig
     """
+    matching_jobs = []
```

Comment:
```suggestion
    matching_jobs = set()
```

Reply:
I already have the set there, but I am not sure if we don't want to preserve order. Can this be useful? Now, I can't remember the usecase for that...;)